### PR TITLE
Help Center: Home & Navigation

### LINKED ISSUE #965: [Help Center] Homepage and Publisher Navigation
Created designs for the homepage and publisher navigation page for the help center.

[Figma](https://www.figma.com/file/dvX3yg7FMGJSWFaqHWrWgQ/Justice-Counts-Publisher?type=design&node-id=9299%3A14912&mode=design&t=yiYUFIqWFt30GJKq-1)

### PR DESCRIPTION
**Pull request by mxosman**
## Description of the change

Implements newly designed homepage and guide directory navigation.

Reference: [Figma](https://www.figma.com/file/dvX3yg7FMGJSWFaqHWrWgQ/Justice-Counts-Publisher?type=design&node-id=9299%3A14912&mode=design&t=yiYUFIqWFt30GJKq-1)
Playtesting Link: https://mahmoudtest---justice-counts-web-qqec6jbn6a-uc.a.run.app/help

## Related issues

Closes #965

### Development

**This box MUST be checked by the submitter prior to merging**:
- [x] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [ ] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: publisher/src/components/HelpCenter/HelpCenterDirectory.tsx [hunk 1]
```diff
@@ -19,8 +19,28 @@ import { groupBy } from "@justice-counts/common/utils";
...
+    <Styled.Thumbnail src={publisherThumbnail} alt="" />
+  ),
+  [GuideCategories.AdvancedConcepts]: (
+    <Styled.Thumbnail src={publisherThumbnail} alt="" />
+  ),
+  [GuideCategories.InteractWithTheData]: (
+    <Styled.Thumbnail src={publisherThumbnail} alt="" />
+  ),
+  [GuideCategories.Dashboards]: (
+    <Styled.Thumbnail src={publisherThumbnail} alt="" />
```
Comment (mxosman) [inline publisher/src/components/HelpCenter/HelpCenterDirectory.tsx:41]: These will be replaced with the proper thumbnail images.

File: publisher/src/components/HelpCenter/HelpCenterSetup.tsx [hunk 2]
```diff
@@ -18,32 +18,47 @@
...
+import { ReactComponent as AgencySettingsIcon } from "../assets/hc-agency-settings-icon.svg";
+import { ReactComponent as AutomatedBulkUploadIcon } from "../assets/hc-automated-bulk-upload-icon.svg";
+import { ReactComponent as BulkUploadIcon } from "../assets/hc-bulk-upload-icon.svg";
+import dashboardThumbnail from "../assets/hc-dashboard-help-guide-thumbnail.png";
+import { ReactComponent as ExploreDataIcon } from "../assets/hc-explore-your-data-icon.svg";
+import { ReactComponent as ManualEntryIcon } from "../assets/hc-manual-entry-icon.svg";
+import publisherThumbnail from "../assets/hc-publisher-help-guide-thumbnail.png";
+import { ReactComponent as SetUpMetricsIcon } from "../assets/hc-set-up-metrics-icon.svg";
+// import { ReactComponent as SuperagenciesIcon } from "../assets/hc-superagencies-icon.svg";
+// import { ReactComponent as SupervisionDisaggregationsIcon } from "../assets/hc-supervision-disaggregations-icon.svg";
```
Comment (mxosman) [inline publisher/src/components/HelpCenter/HelpCenterSetup.tsx:30]: Whenever we have content for these sections, these can be uncommented and used.

**Comment by mxosman** [comment]
Hi @forrestkim00! I wasn't sure if there was any desired behavior with the category box thumbnails - I mimicked what was in the homepage, but let me know if you'd rather them be static and have no hover behavior. Here's what I'm referring to:

https://github.com/Recidiviz/justice-counts/assets/59492998/313bb0a8-8477-4b42-820d-2c137e6221c8

**Comment by jwhrec** [comment]
I would prefer static but happy to defer to @forrestkim00

**Comment by forrestkim00** [comment]
Hey Mahmoud! Let's keep the thumbnails static, I think it helps differentiate the components from the previous page. Thanks for clarifying!

**Comment by mxosman** [comment]
Awesome - thank you both for the clarification! Will ping when this is redeployed!

**Comment by mxosman** [comment]
Changes have been redeployed!

File: publisher/src/components/HelpCenter/HelpCenterSetup.tsx (hunk 2 above)
Comment (lilidworkin) [reply publisher/src/components/HelpCenter/HelpCenterSetup.tsx:30]: Do we not have content for these? I thought we should. cc @jwhrec

**Comment by jwhrec** [comment]
(sorry I didn't know how to reply in thread)

- Will Dashboard always be the only page in the Dashboards category? If so maybe we can just go straight to the content?

Yes let's just go straight to content here for now, we can bring this back as new pages get created. 

- It was unclear to me at first that I could use the breadcrumb to nav back. I think because it's light grey so it looks unclickable. Maybe it could underline on hover or something like that?

This works for me!

- Do we not have content for these? I thought we should. cc @jwhrec

We do, I just want to get edits finalized before moving content over!


Also another thing I noticed. We will want to add help center to the top right hand corner drop down. Let's do it as the second lowest option right above log out.

File: publisher/src/components/HelpCenter/HelpCenterSetup.tsx (hunk 2 above)
Comment (jwhrec) [reply publisher/src/components/HelpCenter/HelpCenterSetup.tsx:30]: We should cc @forrestkim00 - can you provide the icons for these?

Comment (mxosman) [reply publisher/src/components/HelpCenter/HelpCenterSetup.tsx:30]: Last time I checked it was still in TBD - but, I can add what's in the doc now if that helps.

**Comment by mxosman** [comment]
> > (sorry I didn't know how to reply in thread)
> > 
> > * Will Dashboard always be the only page in the Dashboards category? If so maybe we can just go straight to the content?
> > 
> > Yes let's just go straight to content here for now, we can bring this back as new pages get created.
> 
> Ah, hmm - I didn't quite think about the possibility of going directly from the home to a guide. Let me think some more about what I need to do to accommodate this!
>
Just to follow up - this will be a bit tricker than I thought, I'll need rethink some pieces of this structure so I'll take care of this in a follow up PR.

> > * It was unclear to me at first that I could use the breadcrumb to nav back. I think because it's light grey so it looks unclickable. Maybe it could underline on hover or something like that?
> > 
> > This works for me!
 
> > Also another thing I noticed. We will want to add help center to the top right hand corner drop down. Let's do it as the second lowest option right above log out.
> 
> I wasn't sure if the entry point was determined yet so I haven't added anything. Will add this to the Publisher menu!

@forrestkim00 - is there a good icon to use for this in the menu (I just reused the Your Account icon)?

Here's how it looks now (redeploying to playtesting soon):

https://github.com/Recidiviz/justice-counts/assets/59492998/60f50c3b-b810-4ee9-a971-348e8c608258

**Comment by forrestkim00** [comment]
Looking great! 😁 Just dropped an icon in the [appendix](https://www.figma.com/file/dvX3yg7FMGJSWFaqHWrWgQ/Justice-Counts-Publisher?type=design&node-id=9601%3A3875&mode=design&t=19RbmDi3PTIt0au6-1)!

image

**Comment by mxosman** [comment]
> > > (sorry I didn't know how to reply in thread)
> > > 
> > > * Will Dashboard always be the only page in the Dashboards category? If so maybe we can just go straight to the content?
> > > 
> > > Yes let's just go straight to content here for now, we can bring this back as new pages get created.
> > 
> > 
> > Ah, hmm - I didn't quite think about the possibility of going directly from the home to a guide. Let me think some more about what I need to do to accommodate this!
> 
> Just to follow up - this will be a bit tricker than I thought, I'll need rethink some pieces of this structure so I'll take care of this in a follow up PR.

@lilidworkin & @jwhrec - what do you think of this solution? I tried a number of ways and this one requires the least refactoring. It will link directly to the guide from the home page, but are we OK with the name of the app still appearing in the breadcrumbs if I disable any hover behavior on it? 

https://github.com/Recidiviz/justice-counts/assets/59492998/4f4d1dbb-4255-4ba3-af08-ed92f08ffe96

**Comment by mxosman** [comment]
> Looking great! 😁 Just dropped an icon in the [appendix](https://www.figma.com/file/dvX3yg7FMGJSWFaqHWrWgQ/Justice-Counts-Publisher?type=design&node-id=9601%3A3875&mode=design&t=19RbmDi3PTIt0au6-1)!
> 
> image

Amazing! Thank you so much, Forrest!

**Comment by jwhrec** [comment]
This solution works for me, thanks Mahmoud!

**Comment by mxosman** [comment]
> This solution works for me, thanks Mahmoud!

Amazing! Thank you, Jessica!

Here it is redeployed incase anyone wants to play with it so far: https://mahmoudtest---justice-counts-web-qqec6jbn6a-uc.a.run.app/

I'm going to merge these changes in now, but certainly happy to follow up on any additional feedback!

**Pull request merged**